src/index.ts

```diff
@@ -22,38 +28,43 @@ function createButtons() {
         disabled: disabled.texture,
     };
 
-    const label = {
-        text: "Label text",
-        style: {
-            fill: "#fff",
-            fontSize: "26px",
-            fontFamily: "Arial, sans-serif",
-        },
-    };
+    const button = new Button({textures});
+    button.anchor.set(0.5);
+    button.x = app.view.width / 2;
+    button.y = app.view.height - button.height - 50;
+
+    return button;
+}
+
+function createSymbol() {
+    const symbolSize = 100;
 
-    const button = new Button({
-        textures,
-        label,
-    });
+    const symbol = new Symbol(
+        Loader.shared.resources[`symbol-${getRandomInt(1, 3)}`].texture,
+        app.view.width / 2,
+        symbolSize * 3,
+        symbolSize
+    );
 
-    app.stage.addChild(button);
+    return symbol;
+}
 
-    const button2 = new Button({textures});
-    button2.x = 250;
-    button2.y = 250;
-    button2.disable();
+function init() {
+    const button = createButton();
+    const symbol = createSymbol();
 
-    setTimeout(() => {
-        button2.enable();
-    }, 2500);
+    button.setCallback(onButtonClick);
 
-    app.stage.addChild(button2);
+    app.stage.addChild(button, symbol);
 
-    const button3 = new Button({textures});
-    button3.x = 450;
-    button3.y = 150;
+    function onButtonClick() {
```

Comment:
consider using async/await functions for this (the "sync" syntax for async code - in most cases it's more convenient):
```
async function init() {
    //...
    async function onButtonClick() {
        this.disable()
        await symbol.startBounce();
        await symbol.moveOneSlot();
        await symbol.moveOneSlot();
        await symbol.moveOneSlot();
        await symbol.endBounce();
        this.enable();
    }
}
```